### PR TITLE
rework sponsor images

### PR DESCRIPTION
**Pull request by ianmah**
I made all the images smaller because they looked too big to me.

The main bugs were caused by missing `max-height` and `top: -100` not being supported but I also trimmed down the code a bit.

Desktop
image

mobile stacks
image

### CODE REVIEW

File: components/SponsorImage.vue [hunk 1]
```diff
@@ -36,12 +36,18 @@ export default {
 <style scoped>
 .default {
   transition: 0.5s, -moz-filter 0.5s, -o-filter 0.5s, filter 0.5s;
-  width: 100%;
   filter: brightness(0) invert(1);
 }
+img {
+  width: 100%;
```
Comment (ianmah) [inline components/SponsorImage.vue]: before, only the non-alt image had this set which caused inconsistencies (and very weird wrapping of alt images)

File: components/SponsorImage.vue [hunk 2]
```diff
@@ -36,12 +36,18 @@ export default {
...
+}
 .default:hover {
   filter: none;
 }
+.link {
+  display: block;
```
Comment (ianmah) [inline components/SponsorImage.vue]: an elemtnt (alt image) will position itself "absolute" relative to the closest block element, so make the closest block element the link element

File: components/SponsorImage.vue [hunk 3]
```diff
@@ -52,7 +58,7 @@ export default {
   opacity: 1;
 }
 .link .altImage {
-  top: -100;
```
Comment (ianmah) [inline components/SponsorImage.vue]: `-100` is really buggy

File: components/Sponza.vue [hunk 4]
```diff
@@ -4,71 +4,33 @@
       Sponsors
     </p>
     <br>
-    <!--  -->
-    <div id="sponsors-list" :class="is-flex">
-      <div class="columns is-multiline is-vcentered is-centered">
-        <div v-for="item in items" :key="item.name">
-          <div v-if="item.rank == 'tera'">
-            <div class="column">
```
Comment (ianmah) [inline components/Sponza.vue]: Was this intentionally using buefy styles?

Comment (Ciuffi) [reply components/Sponza.vue]: I'm pretty sure it was

**Pull request merged**